schema/dcschema_enum_classes.mcf

```diff
@@ -383,3 +383,8 @@ typeOf: schema:Class
 subClassOf: schema:Enumeration
 name: "MedicalStatusEnum"
 description: "Medical status of a medical incident, indicating whether the patient is hospitalized, tested, deceased, etc."
+
+Node: dcid:MedicalTestResultTypeEnum
```

Comment:
Can probably get rid of "Type" (note that this does not need data reimport. I'm also ok if you don't reimport the schema to dev.)

Reply:
Will do it :)